src/main/java/com/senkiv/bookstore/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,23 @@
+package com.senkiv.bookstore.dto;
+
+import com.senkiv.bookstore.validation.annotation.PasswordMatch;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+@PasswordMatch
```

Comment:
better make this annotation more abstract and reuse it in the future  (FieldMatcher)